src/app/views/marketplace/marketplace-compare.controller.coffee

```diff
@@ -0,0 +1,33 @@
+angular.module 'mnoEnterpriseAngular'
+  .controller('DashboardMarketplaceCompareCtrl', ($scope, $stateParams, $state,
+    MnoeMarketplace, PRICING_CONFIG) ->
+
+      vm = this
+
+      vm.isPriceShown = PRICING_CONFIG && PRICING_CONFIG.enabled
+
+      #====================================
+      # Initialization
+      #====================================
+      vm.isLoading = true
+
+      currency = (PRICING_CONFIG && PRICING_CONFIG.currency) || 'AUD'
+      vm.pricing_plans = [currency] || 'AUD' || 'default'
```

Comment:
@x4d3 is it correct that we are going to display plans only the plans for one currency, not for all?